src/gui/test/__snapshots__/routes.tsx.snap

```diff
@@ -85,8 +85,6 @@ exports[`renders Layout for the "/help" view 1`] = `
     <gui-sidebar-inset>
       <gui-header>
       </gui-header>
-      <gui-help-selectors>
-      </gui-help-selectors>
```

Comment:
is this change intentional? 🤔

Reply:
That happened when I ran tests at the top level, I'm trying to figure out why it happened and why it's failing.

But no.